### PR TITLE
Pvwc mental health assessment

### PR DESCRIPTION
**Pull request by abikarmakar**

### CODE REVIEW

File: models/intermediate/pvwc/int_pvwc_models.yml
Comment (vinod-rajasekaran) [inline models/intermediate/pvwc/int_pvwc_models.yml:1]: can we remove commented lines to make this cleaner

File: models/intermediate/pvwc/pvwc_phq_gad_normalized.sql
Comment (vinod-rajasekaran) [inline models/intermediate/pvwc/pvwc_phq_gad_normalized.sql:1]: can we remove unused fields here like reg_id, full name, centre etc that we are no pulling form case file ; also if not needed lets not import phq1-8 and gad1-8

File: models/production/pvwc/raw/pvwc_phq_gad.sql
Comment (vinod-rajasekaran) [inline models/production/pvwc/raw/pvwc_phq_gad.sql:1]: more descriptive cte names please.

not understanding the two row numbers in cte a

why integer grades and then change them back to string again ?

we need to sit together and review - not super easy for me to look at

**Review by vinod-rajasekaran — CHANGES_REQUESTED** [review]

File: models/intermediate/pvwc/pvwc_phq_gad_normalized.sql
Comment (abikarmakar) [reply models/intermediate/pvwc/pvwc_phq_gad_normalized.sql:1]: Done.

File: models/intermediate/pvwc/int_pvwc_models.yml
Comment (abikarmakar) [reply models/intermediate/pvwc/int_pvwc_models.yml:1]: Done.

File: models/production/pvwc/raw/pvwc_phq_gad.sql
Comment (abikarmakar) [reply models/production/pvwc/raw/pvwc_phq_gad.sql:1]: cte's given descripive names.

**Review by vinod-rajasekaran — CHANGES_REQUESTED** [review]

File: models/intermediate/pvwc/pvwc_phq_gad_normalized.sql
[the remark this answers is not in the document]
Comment (abikarmakar) [reply models/intermediate/pvwc/pvwc_phq_gad_normalized.sql:1]: Done

[the remark this answers is not in the document]
Comment (abikarmakar) [reply models/intermediate/pvwc/pvwc_phq_gad_normalized.sql:1]: Done.

File: models/intermediate/pvwc/pvwc_int_phq_gad.sql
[the remark this answers is not in the document]
Comment (abikarmakar) [reply models/intermediate/pvwc/pvwc_int_phq_gad.sql:1]: Done.

**Review by vinod-rajasekaran — APPROVED** [review]

**Pull request merged**